com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JdtSourceLookUpProvider.java

```diff
@@ -120,11 +119,7 @@ public boolean supportsRealtimeBreakpointVerification() {
         String filePath = AdapterUtils.toPath(uri);
         // For file uri, read the file contents directly and pass them to the ast parser.
         if (filePath != null && Files.isRegularFile(Paths.get(filePath))) {
-            Charset cs = (Charset) this.options.get(Constants.DEBUGGEE_ENCODING);
-            if (cs == null) {
-                cs = Charset.defaultCharset();
-            }
-            String source = readFile(filePath, cs);
+            String source = readFile(filePath);
```

Comment:
When parsing Java source files, let's keep using the same encoding as JDT language server.